src/genome.py

```diff
@@ -109,8 +116,18 @@ def insert_te(self, pos: int, length: int) -> int:
 
         Returns a new ID for the transposable element.
         """
-        ...  # FIXME
-        return -1
+        if len(self.TE)==0: 
+            ID = 1
+        else: 
+            ID = max(self.TE) + 1
```

Comment:
If you keep all the TE IDs you've ever seen in `self.TE` then `len(self.TE) + 1` should always work (no need for a special case). If you don't, then getting `max(self.TE)` let's you recycle identifiers (which I don't think is a problem in this application, but generally will be).